config/locales/de.yml

```diff
@@ -217,23 +373,191 @@ de:
-    thanks_for_interest: Vielen Dank für Ihr Interesse an einer Teilname an unserer Veranstaltung, %{title}.
+    thanks_for_interest: Vielen Dank für Ihr Interesse an einem Beitrag zu unserer Veranstaltung, %{title}.
+    to: zu
     update: Speichern
     update_availability_notice: Vielen Dank, dass Sie Ihre Verfügbarkeit angegeben haben.
     updated: Erfolgreich gespeichert.
     uploaded_files: Hochgeladene Dateien
     welcome: Willkommen
     welcome_text: Willkommen beim Call for Participation. Zum Erstellen oder Bearbeiten von Einreichungen muss man eingeloggt sein. Wenn Sie hier neu sind, müssen Sie sich kurz registrieren.
     withdraw: zurückziehen
-    withdrawal_confirmation: Wollen Sie diesen Vorschlag wirklich zurückziehen?
+    withdrawal_confirmation: Wollen Sie diese Einreichung wirklich zurückziehen?
   classifiers: Classifiers
+  classifiers_module:
+    current_classifiers: 'Die aktuell eingerichteten Classifiers sind:'
+    empty_classifiers: |
+      Hier können Sie die Classifiers einrichten und bearbeiten, die Einreicher
+      ihren Events vergeben können.
+    modify_classifier_with_parent: |
+      Die Classifiers für diese Konferenz können nur von Adminstratoren der
+      Parent-Konferenz %{parent} verändert werden.
+  clear: löschen
+  col_abstract: 'Abstrakt:'
+  col_availability: 'Verfügbarkeit:'
+  col_avg_audience_feedback: 'Durchschnittliches Feedback der Zuschauer:'
+  col_classifiers: 'Classifiers:'
+  col_col_template: "%{data}:"
+  col_conference: 'Konferenz:'
+  col_description: 'Beschreibung:'
+  col_email: 'E-Mail:'
+  col_email_for_login: 'E-Mail für die Anmeldung:'
+  col_event_state: Event-Status
+  col_files: 'Dateien:'
+  col_filters: 'Filter:'
+  col_first_name: 'Vorname:'
+  col_full_name: 'Vollständiger Name:'
+  col_gender: 'Geschlecht:'
+  col_language: 'Sprache:'
+  col_last_login: 'Letzte Anmeldung:'
+  col_last_name: 'Nachname:'
+  col_links: 'Links:'
+  col_non_reimbursed_expenses: 'Nicht erstattete Spesen:'
+  col_notes_admin: 'Notizen (admin):'
+  col_phone_number_with_type: 'Telefonnummer (%{type}):'
+  col_public: 'öffentlich:'
+  col_public_name: 'Öffentlicher Name:'
+  col_reimbursed_expenses: 'Zurückerstattete Spesen:'
+  col_role: 'Rolle:'
+  col_room: 'Raum:'
+  col_rooms: 'Räume:'
+  col_speakers: 'Sprecher:'
+  col_start_time: 'Startzeit:'
+  col_submission_notes: 'Einreichungshinweise (Benutzer und Admin):'
+  col_ticket: 'Ticket:'
+  col_time_slots: 'Dauer (in Slots):'
+  col_total: 'Gesamt:'
+  col_track: 'Track:'
+  col_tracks: 'Tracks:'
+  col_type: 'Typ:'
+  comment: Kommentar
+  comments: Kommentare
+  conference: Konferenz
+  conference_management: Konferenzmanagement
+  conferences: Konferenzen
+  conferences_module:
+    accepting: accepting
+    add_conference: Konferenz hinzufügen
+    add_conference_hint: Fügen Sie eine neue Konferenz hinzu
+    canceled: canceled
+    conference_in_past: Die Konferenz liegt in der Vergangenheit. Keine Einreichungen sind möglich.
```

Comment:
besser: "Die Konferenz liegt in der Vergangenheit, der sind keine Einreichungen mehr möglich."